src/OneExplorer.ts

```diff
@@ -65,30 +65,41 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     }
   }
 
-  // TODO(dayo): enable refresh command
   refresh(): void {
     this._onDidChangeTreeData.fire();
   }
 
   getTreeItem(element: OneNode): vscode.TreeItem {
-    element
-        .command = {command: 'oneExplorer.openFile', title: 'Open File', arguments: [element.node]};
     return element;
   }
 
   getChildren(element?: OneNode): OneNode[]|Thenable<OneNode[]> {
-    if (this.oneTree === undefined) {
-      vscode.window.showInformationMessage('No ONE model or config in empty workspace');
-      return Promise.resolve([]);
+    if (!element) {
+      if (!this.workspaceRoot) {
+        vscode.window.showInformationMessage(`Cannot find workspace root.`);
+        return Promise.resolve([]);
+      } else {
+        return Promise.resolve(this.getNode(this.getTree(this.workspaceRoot)));
```

Comment:
Root tree must be generated for every getChildren() calls so that the tree get actually refresehd (FYI, getChilderen calls for the first time you open the collapsed tree)